### PR TITLE
Implement bottom panel

### PR DESCRIPTION
**Pull request by steelbrain**
Looks like this on 18 Sep:

- [x] Implement panel
- [x] Implement sorting
- [ ] Implement resize
- [x] Implement proper hiding/showing appropriately
- [x] Do something about multi-line error messages and their css to make sure they don't go crazy

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
 {
   "extends": "steelbrain",
+  "plugins": ["react"],
   "rules": {
     "no-param-reassign": 0,
     "no-duplicate-imports": 0,
-    "no-underscore-dangle": 0
+    "no-underscore-dangle": 0,
```
Comment (Arcanemagus) [inline .eslintrc.json]: `0` -> `"off"`

File: src/panel/element.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        style={{ width: '100%' }}
+        className='linter'
+      />
+    )
+  }
+  static renderRowColumn(row: LinterMessage, column: string): string | Object {
+    switch (column) {
+      case 'file':
+        if (row.version === 1) {
+          return row.filePath ? atom.project.relativizePath(row.filePath)[1] : ''
```
Comment (Arcanemagus) [inline src/panel/element.js]: This is the first method.

File: src/panel/element.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      if (sortColumns.linterName) {
+        const multiplyWith = sortColumns.linterName === 'asc' ? 1 : -1
+        const sortValue = a.severity.localeCompare(b.severity)
+        if (sortValue !== 0) {
+          return multiplyWith * sortValue
+        }
+      }
+      if (sortColumns.file) {
+        const multiplyWith = sortColumns.file === 'asc' ? 1 : -1
+        const fileA = atom.project.relativizePath(a.version === 1 ? (a.filePath || '') : a.location.file)[1].length
```
Comment (Arcanemagus) [inline src/panel/element.js]: Here being the second method.

**Comment by steelbrain** [comment]
I tried to take on the resize part with `interact.js` the same library we're using in Linter v1, but it doesn't work with React.

Therefore, I'm gonna merge this as-is for now and work on it afterwards

**Pull request merged**